### DISCUSSION TITLE
"Failed to load PicoGK Library"

### DISCUSSION THREAD

**Discussion by Ewicko** [leap71/PicoGK#10] [Q&A]
Hello, I'm having trouble running the test boolean in the README file. 

I'm getting the error "Failed to load PicoGK Library" on line 175 of the PicoGK_Library.cs file that pops up after running Program.cs

Is this common? Any fix that you can think of? 

Some context: 
I'm on Mac
When I downloaded the file from the installation guide, it doesn't seem like it was zipped and it simply downloaded a folder called "PicoGK-main". This is the folder that I copied to the project folder with Program.cs in it. 
The installation guide says move the "files" to usr/local/lib but I only found one file in my folder I downloaded (the one called "PicoGK-main").

Thanks for the help!

**Comment by LinKayser** [comment]
Are you downloading this one? https://github.com/leap71/PicoGK/archive/refs/tags/v1.0.0.zip

This is definitely a ZIP, so double click to unpack it.

Also, I just put an ARM-based Mac installer into a forum post over here: https://github.com/leap71/PicoGK/discussions/11

This greatly simplifies the setup of the runtime on the Mac. If you install this, you should be able to test out the examples immediately from Visual Studio.

Let me know if this helps.
Lin
